base/ltoutput.dtx

```diff
@@ -2592,12 +2653,62 @@
 \NewSocketPlug {build/column/outputbox}{footnotes-floats-legacy} {%
 %    \end{macrocode}
 %    In the legacy case we don't really want to take out the bottom
-%    skip, but rather then altering \cs{@outputbox@removebskip} in
+%    skip, but rather than altering \cs{@outputbox@removebskip} in
 %    \cs{makecol} to do nothing, which we would then have to undo in
 %    each other layout, we simply reinsert the dropped skip
```

Comment:
each  ==>  every 

And Better:
  . . . layout, we immediately reinsert the dropped skip.